src/main/java/com/exadel/discountwebapp/location/controller/LocationController.java

```diff
@@ -0,0 +1,48 @@
+package com.exadel.discountwebapp.location.controller;
+
+import com.exadel.discountwebapp.location.entity.Location;
+import com.exadel.discountwebapp.location.service.LocationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/location")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class LocationController {
+    LocationService locationService;
+
+    @GetMapping("/")
+    private List<Location> getAllLocations() {
```

Comment:
Replace list of the Location with list of the LocationResponseVO (below too)

Reply:
done